assets/js/modules/analytics/setup.js

```diff
@@ -95,6 +95,9 @@ class AnalyticsSetup extends Component {
 
 				if ( existingTag && existingTag.length  ) {
 					try {
+
+						// Verify user has access to found tag.
+						await data.get( 'modules', 'analytics', 'tag-permission', { tag: existingTag }, false );
```

Comment:
Please update/reword this comment to indicate that an error will be thrown if the user does not have access to the tag. This is implied by the code since there is no return value used, but let's be explicit about that since it isn't obvious from the code itself. Thanks!